javascript-timers/interval.js

```diff
@@ -0,0 +1,21 @@
+$countdown = document.querySelector('.countdown-display')
+
+let interval;
+
+function settingInterval(callback) {
+  if ($countdown.textContent === '~Earth Beeeelooowww Us~') {
+   clearInterval(interval);
+   return;
+  }
+  if ($countdown.textContent === '4') {
+    $countdown.textContent = '3'
+  } else if ($countdown.textContent === '3') {
+    $countdown.textContent = '2'
+  } else if ($countdown.textContent === '2') {
+    $countdown.textContent = '1'
+  } else {
+    $countdown.textContent = '~Earth Beeeelooowww Us~'
+  }
+}
```

Comment:
We don't want to hard code values in the conditional. We should be utilizing a count variable to keep track of the current number, and let that dictate our `textContent`